src/gc/gcinterface.h

```diff
@@ -205,6 +205,41 @@ extern uint8_t* g_shadow_lowest_address;
 // For low memory notification from host
 extern int32_t g_bLowMemoryFromHost;
 
+// Event levels corresponding to events that can be fired by the GC.
+enum GCEventLevel
+{
+    GCEventLevel_None = 0,
+    GCEventLevel_Fatal = 1,
+    GCEventLevel_Error = 2,
+    GCEventLevel_Warning = 3,
+    GCEventLevel_Information = 4,
+    GCEventLevel_Verbose = 5,
+    GCEventLevel_Max = 6
+};
+
+enum GCEventKeyword
+{
+    GCEventKeyword_None                          =       0x0,
```

Comment:
This information needs to be kept in sync with the manifest, which is currently the one source of truth for all events.

At a minimum, there should be a comment indicating this, but ideally, this information is pulled from the manifest and generated.  Adding new events won't touch this, but if you want to modify the set of keywords you'll have to know to touch this.

Reply:
good point - I'll call this out in a comment.